app.json

```diff
@@ -6,7 +6,7 @@
 
```

Comment:
we remove these config fields - `addons`, `buildpacks`, `formation` and `scripts` since there are no values for them.

Reply:
okay I am removing this now

Reply:
this has been resolved @Japhethca